__tests__/commands/authOptions.test.ts

```diff
@@ -0,0 +1,78 @@
+import { program, CommanderStatic } from 'commander';
+import { afterAll, beforeAll, describe, expect, test } from '@jest/globals';
+import authCommand from '../../src/commands/Auth/authCommand';
+import {
+  createTempDir,
+  createTemplate,
+  initProject,
+  linkDirCustom,
+  removeTempDir
+} from '../test-helpers';
+
+const PROJECT_NAME = 'test-project';
+const ORIGINAL_CWD = linkDirCustom();
+const TEMP_DIR = ORIGINAL_CWD + '/authOptionsTemp';
+const AUTH_OPTIONS = [
+  'node',
+  'auth',
+  '--app-name',
+  'test-app',
+  '--domain',
+  'https://domain.kintone.com',
+  '--app-id',
+  '100',
+  '--username',
+  'user',
+  '--password',
+  'password',
+  '--use-proxy',
+  '--proxy',
+  'http://localhost:8080'
+];
+
+describe('auth command: options', () => {
+  let mainProgram: CommanderStatic;
+
+  beforeAll(async () => {
+    createTempDir(TEMP_DIR);
+
+    await initProject(TEMP_DIR, PROJECT_NAME);
+    await createTemplate(TEMP_DIR, PROJECT_NAME);
+
+    mainProgram = authCommand(program);
+    process.argv = AUTH_OPTIONS;
+    await mainProgram.parseAsync(process.argv);
+  });
+
+  afterAll(() => {
+    removeTempDir(TEMP_DIR);
+  });
+
+  test('should have appName as "test-app"', async () => {
+    expect(mainProgram.opts().appName).toBe('test-app');
```

Comment:
I think mainProgram.opts() came from the other source code.
I think unit test should be separated from the other source code.
Please modify it.

Other parts are the same.

Reply:
How about this?